app.go

```diff
@@ -244,24 +246,31 @@ func pushImage(image string) {
 	execBuild("./", []string{"docker", "push", image})
 }
 
-func buildImage(image string, handler string, functionName string, language string, nocache bool) {
+func buildImage(image string, handler string, functionName string, language string, nocache bool, squash bool) {
 
 	switch language {
 	case "node", "python":
 		tempPath := createBuildTemplate(functionName, handler, language)
 
 		fmt.Printf("Building: %s with Docker. Please wait..\n", image)
 
-		cacheFlag := ""
+		//lets create a buffer for any build flags
+		var flagBuf bytes.Buffer
```

Comment:
I thought that would be the thinking. This code is building a very short string so I think I prefer the string concatenation and readability.